src/script/lua_api/l_server.cpp

```diff
@@ -539,4 +717,11 @@ void ModApiServer::Initialize(lua_State *L, int top)
 
 	API_FCT(get_last_run_mod);
 	API_FCT(set_last_run_mod);
+
+	API_FCT(auth_read);
```

Comment:
Can you make auth be a specific Lua object instead of a static function API ? It can be better to have an auth object to manipulate

Reply:
define a new C++ object, reference it with a LuaRef on the Server stack at initialization like it's done for CSM with "ui" namespace